models/shopify__customer_cohorts.sql

```diff
@@ -1,9 +1,28 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='customer_cohort_id',
+        incremental_strategy='insert_overwrite' if target.type in ('bigquery', 'databricks') else 'delete+insert',
```

Comment:
This is an interesting question. I think it would be best to avoid any circumstance in which we would be removing functionality from current Databricks users. From that perspective would it be possible to identify any users that are primarily using Databricks SQL, and then apply a different version of the package? Another way to approach this would be to understand if we have access via Quickstart to see if the user is using Databricks Runtime or Databricks SQL, and apply a different version of the package.